pytorch_lightning/tuner/lr_finder.py

```diff
@@ -62,24 +61,160 @@ def _determine_lr_attr_name(trainer, model: LightningModule) -> str:
     )
 
 
+class _LRFinder(object):
```

Comment:
Note to reviewers: This class was moved as we need `_LRFinder` to be defined for the `lr_find` signature right below